packages/fast-components-react-base/src/typography/typography.tsx

```diff
@@ -49,9 +49,9 @@ class Typography extends Foundation<ITypographyHandledProps & IManagedClasses<IT
      */
     protected generateClassNames(): string {
         const classes: string = this.props.typeLevel ?
-            get(this.props, `managedClasses.typography_${this.props.typeLevel}`) : get(this.props, "managedClasses.typography_1");
+            get(this.props, `managedClasses.typography__${this.props.typeLevel}`) : get(this.props, "managedClasses.typography__1");
 
-        return super.generateClassNames(classes);
+        return super.generateClassNames(`${get(this.props, "managedClasses.typography")} ${classes}`);
```

Comment:
for consistency in how this is typically done throughout the repo would prefer the base class be set first...but that's more preference. This actually leads to an interesting question - is level required now that we have a base class for users to hook to?

Reply:
It is? `super.generateClassNames(`${get(this.props, "managedClasses.typography")} ${classes}`);`
The typography base class will always be set first... if we decide we don't need a default, this syntax would change.

Reply:
Yeah - I see that. Multitasking when I wrote.

I would propose though that even if we do it in a different task, we should not set a default type level on the base component if we have a managed class to target the base.